tests/parser/ScheduleTests.cpp

```diff
@@ -1338,6 +1338,7 @@ DATES             -- 1
 )";
 
     const auto& schedule = make_schedule(input);
+    static_cast<void>(schedule); // Silence unused variable warning.
```

Comment:
Just remove the entire test - it is not interesting. In the case the test actually makes sense to have I usually implement it with `BOOST_CHECK_NO_THROW()`:
```C++
BOOTS_CHECK_NO_THROW( make_schedule( input ) );
```